elasticsearch/security_api.go

```diff
@@ -104,10 +85,6 @@ type GetAPIKeyResponse struct {
 	APIKeys []APIKeyResponse `json:"api_keys"`
 }
 
-type CreatePrivilegesRequest map[AppName]PrivilegeGroup
-
-type CreatePrivilegesResponse map[AppName]PrivilegeResponse
```

Comment:
I think [`PrivilegeResponse` and `PrivilegeGroup`](https://github.com/elastic/apm-server/pull/3251/files#diff-39e6b7ff4f3bd5569c98e478c2740cdbR138-R140) is also unused now

Reply:
good catch, somehow my IDE didn't flag it